CMakeLists.txt

```diff
@@ -29,3 +31,4 @@ LIST(APPEND WASM_INCLUDES ${WW_COMMON_INCLUDES})
 ENABLE_TESTING()
 
 ADD_SUBDIRECTORY(exchange-contract)
+ADD_SUBDIRECTORY(digital-asset-contract)
```

Comment:
Could you explain what you mean? There is no actionable request here & i don't understand why this is related to the CMake files.  What dependencies are you concerned about?

The contract packages specify a dependency on a particular PDO client version (which may or may not be built from the submodule, the test is against the virtual environment at install time NOT against the submodule at build time). 

We must ensure that some predictable PDO version works for the contract families & it is available. The contract build code must have the PDO source because the contract code depends on the includes in the pdo contracts/wawaka repository. We may choose to break out the contract build code in the future, but for now it must be included. 

I completely disagree that it is unrealistic to expect all of the contract families to work with a specific version of the PDO client and contract build packages.  And in fact when we add automated testing that will be precisely the requirement. If you add a contract family, then you accept the responsibility to keep it up to date with the version of PDO that is included in the contracts repository. Any contract family that fails to pass this test should not be included.